### PR TITLE
Access tweaks

### PR DESCRIPTION
**Pull request by TheUltimateNuke**
just made a bunch of things readonly when suggested and made some small improvements to stability on the way.
oh, i also removed that MMHOOK thing in /ref/ since we weren't using it

### CODE REVIEW

File: src/Cement.cs [hunk 1]
```diff
@@ -260,14 +259,14 @@ public void RevertEventSystem()
             _usingCementEventSystem = false;
         }
 
-        public void _Log(object o, BepInEx.Logging.LogLevel logLevel=BepInEx.Logging.LogLevel.Info)
+        private void InternalLog(object o, BepInEx.Logging.LogLevel logLevel=BepInEx.Logging.LogLevel.Info)
```
Comment (HueSamai) [inline src/Cement.cs:268]: This code won't work. You can't have a private void, because then in Log, it won't be able to access InternalLog from Singleton

**Review by HueSamai — DISMISSED** [review]
Check comment on InternalLog

File: src/Cement.cs (hunk 1 above)
Comment (HueSamai) [reply src/Cement.cs:268]: holy crap i was wrong

**Comment by TheUltimateNuke** [comment]
wait so whats the status on this? can i just merge once i sort out conflicts or..?

**Comment by HueSamai** [comment]
> wait so whats the status on this? can i just merge once i sort out conflicts or..?

yeah

**Review by HueSamai — APPROVED** [review]

**Pull request merged**